deepchecks/tabular/context.py

```diff
@@ -58,6 +59,7 @@ class _DummyModel:
     feature_df_list: t.List[pd.DataFrame]
     predictions: pd.DataFrame
     proba: pd.DataFrame
+    classes_: t.List[t.Hashable]
```

Comment:
Check for places where we create dummy models and pass the classes argument when available